core/src/main/java/org/apache/iceberg/StaticTableScan.java

```diff
@@ -26,7 +26,8 @@
 class StaticTableScan extends BaseTableScan {
   private final Function<StaticTableScan, DataTask> buildTask;
 
-  StaticTableScan(TableOperations ops, Table table, Schema schema, Function<StaticTableScan, DataTask> buildTask) {
+  StaticTableScan(TableOperations ops, Table table, Schema schema,
+                  Function<StaticTableScan, DataTask> buildTask) {
```

Comment:
Looks like this didn't need to change.

Reply:
It didn't (was an artifact from a previous refactor), but now with the addition of the `tableType` parameter, it does.